libstuff/STCPNode.h

```diff
@@ -145,6 +144,9 @@ struct STCPNode : public STCPServer {
         static bool isPermafollower(const STable& params);
     };
 
+    // Do we need a mutex protecting this? Depends.
+    list<STCPManager::Socket*> socketList;
```

Comment:
Moved here where it's still used (for now). Because we only use this for sync node peers, we shouldn't need a mutex for accessing it.